components/bluetooth_proxy.rst

```diff
@@ -71,6 +73,8 @@ Below is a complete sample recommended configuration for a Bluetooth proxy. If y
 
 This configuration is for an Olimex ESP32-PoE-ISO board with an Ethernet connection to the network. If you use a different board, you must change the ``board`` substitution to match your board.
 
+For WiFi-based proxies, 1100ms can result in an unstable WiFi connection. Setting the default values for interval and window will usually resolve any instability.
+
```

Comment:
Not sure if we should repeat this?

Instead, maybe change the header to:
```
Complete sample recommended configuration for an ethernet connected Bluetooth proxy
-----------------------------------------------------------------------------------
```

Reply:
adjusted 👍